velox/docs/configs.rst

```diff
@@ -919,3 +911,94 @@ Tracing
      - integer
      - 0
      - The max trace bytes limit. Tracing is disabled if zero.
+
+Async Data Cache (In-Memory Cache)
+----------------------------------
+.. list-table::
+   :widths: 30 10 10 70
+   :header-rows: 1
+
+   * - Property Name
+     - Type
+     - Default Value
+     - Description
+   * - async-data-cache-enabled
+     - bool
+     - true
+     - If true, enable async data cache (in-memory cache).
+   * - cache.velox.ttl-enabled
+     - bool
+     - false
+     - Enable TTL for AsyncDataCache and SSD cache.
+   * - cache.velox.ttl-threshold
+     - string
+     - 2d
+     - TTL duration for AsyncDataCache and SSD cache entries. The following time units are supported: ns, us, ms, s, m, h, d.
+   * - cache.velox.ttl-check-interval
+     - string
+     - 1h
+     - The periodic duration to apply cache TTL and evict AsyncDataCache and SSD cache entries.
+   * - cache.no_retention
+     - bool
+     - false
+     - If true, evict out a query scanned data out of in-memory cache right after the access,
+       and also skip staging to the ssd cache. This helps to prevent the cache space pollution
+       from the one-time table scan by large batch query when mixed running with interactive
+       query which has high data locality. Set the ``hive.node_scheduler_affinity`` session 
+       property to ``NO_PREFERENCE`` to turn cache.no_retention ON. Set the 
+       ``hive.node_scheduler_affinity`` session property to ``SOFT_AFFINITY`` 
+       or ``HARD_AFFINITY`` to turn cache.no_retention OFF.
+
+SSD Cache
```

Comment:
I did not notice some configs apply to both SSD and AsyncDataCache.
Let's use a single table then: `Async Data Cache (In-Memory Cache) and SSD Cache`